app/src/pages/playground/fetchPlaygroundPrompt.ts

```diff
@@ -158,11 +164,22 @@ export const promptVersionToInstance = ({
       __type: "chat",
       messages:
         "messages" in promptVersion.template
-          ? promptVersion.template.messages.map((m) => ({
-              ...m,
-              id: generateMessageId(),
-              role: getChatRole(m.role?.toLocaleLowerCase() as string),
-            }))
+          ? promptVersion.template.messages.map((m) => {
+              const maybeTextPart = m.content.find(isTextPart);
```

Comment:
Maybe leave a comment that you are intentionally only parsing one text part. Otherwise it reads slightly strange